lib/stat_tracker.rb

```diff
@@ -22,37 +24,34 @@ def initialize(games_data, teams_data, game_teams_data)
   end
 
   def highest_total_score
-    Game.all.max_by { |game| game.total_score}.total_score
+    @games_data.max_by { |game| game.total_score}.total_score
   end
 
   def lowest_total_score
-    Game.all.min_by { |game| game.total_score}.total_score
+    @games_data.min_by { |game| game.total_score}.total_score
   end
 
   def biggest_blowout
-    Game.all.max_by { |game| game.margin_of_victory }.margin_of_victory
+    @games_data.max_by { |game| game.margin_of_victory }.margin_of_victory
   end
```

Comment:
Refactoring of Iteration 2 methods to call on instance variable instead of Game class method.